api/managementpb/ia/rules.proto

```diff
@@ -54,38 +54,52 @@ message RuleParam {
     // Bool value.
     bool bool = 3;
     // Float value.
-    float float = 4;
+    double float = 4;
```

Comment:
why data type change is required?

Reply:
When we parse templates YAMLs we use `float64`: https://github.com/percona-platform/platform/blob/c6e7618e5d12abb7419b59e073ee0bc75e5eb20b/pkg/alert/parameter.go#L43

Because of types mismatch between API and underlying structs we had such things: https://github.com/percona/pmm-managed/blob/52a4cd82dda43d4161612f9450964e1474be5574/services/management/ia/common.go#L101-L104